yroots/subdivision.py

```diff
@@ -553,20 +552,20 @@ def getAbsApproxTol(func, deg, a, b):
     """
     tols = []
     np.random.seed(0)
-    for i in range(10):
+    for i in range(5):
```

Comment:
If I'm reading this right, you've changed this to take the average tolerance of 5 randomly chosen spots. I'm guessing that's for speed, right?

Reply:
Right! Cutting the sampling in half from what I saw didn't seem to change the final tolerance very much, but it did speed it up given that we run this function at every level.